src/main/features/response/routes/when-did-you-pay.ts

```diff
@@ -0,0 +1,50 @@
+import * as express from 'express'
+
+import { Paths } from 'features/response/paths'
+
+import { Form } from 'forms/form'
+import { FormValidator } from 'forms/validation/formValidator'
+import { WhenDidYouPay } from 'features/response/form/models/whenDidYouPay'
+import { User } from 'idam/user'
+import { ErrorHandling } from 'common/errorHandling'
+import { Claim } from 'claims/models/claim'
+import { DraftService } from 'services/draftService'
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { Draft } from '@hmcts/draft-store-client'
+
+async function renderView (form: Form<WhenDidYouPay>, res: express.Response, next: express.NextFunction) {
+  try {
```

Comment:
do we need try and catch here as ErrorHandling is surrounding this call with try and catch